queries/earth.jinja2

```diff
@@ -1,78 +1,99 @@
 {% if zoom < 9 %}
 
 SELECT
-    NULL AS name,
-    'earth' AS kind,
-    NULL AS label_placement,
+    '' AS name,
+    sum(way_area)::bigint AS area,
+    mz_calculate_output_earth(t.*) AS mz_properties,
     'naturalearthdata.com' AS source,
     {% filter geometry %}st_union(the_geom){% endfilter %} AS __geometry__,
-    max(gid) AS __id__
+    max(gid) AS __id__,
+    NULL AS label_placement,
+    NULL AS boundary
```

Comment:
Good point, I'll revise.

(This is loosely based off the `water.jinja2` file, probably similar cleanup over there, but I'd rather not mess with that in this PR.)